Add `Tablet Conversions`  and Edit `Upgrading_to_Lazer!` (#14881)
ppy/osu-wiki: wiki/Help_centre/Client/en.md

Later NVidia graphics cards have a global option in the NVidia Control Panel to set the global refresh rate to a setting called 'Fast', which will also fix this issue.

> ### I switched to osu!(lazer)'s native tablet drivers and now my area is not the same! {id=tablet-area}

Reviewer: ```suggestion
### I switched to osu!(lazer)'s native tablet drivers and now my area has changed! {id=tablet-area}
```

> If you were not already using OpenTableDriver's tablet drivers then it is because the tablet you are using is not configured correctly for the native drivers!

Reviewer 2: ```suggestion
If you were not already using OpenTabletDriver's tablet drivers, then it is because the tablet you are using is not configured correctly for the native drivers!
```

> If you were not already using OpenTabletDriver's tablet drivers, then it is because the tablet you are using is not configured correctly for the native drivers!

Reviewer: ```suggestion
osu!(lazer) has the [OpenTabletDriver](https://opentabletdriver.net/) built-in. If you weren't using these drivers before, then this happens because your tablet is not configured correctly for the native drivers.
```

Author: not exactly sure about this one, it feels like the first sentence should have something after the OTD link, but this might just be a case of redundant acronym syndrome and how i have been previously referencing the drivers from OTD.

Author: ill go ahead and add this. Worst that happens is that another review happens from it.

> You can follow [the guide](wiki/Help_centre/Tablet_conversion) for more information on how to correctly convert your old tablet driver's area into OTD's drivers.

Reviewer: ```suggestion
You can follow [this guide](/wiki/Help_centre/Tablet_conversion) for more information on how to correctly convert your old tablet driver's area to OpenTabletDriver's settings.
```

> You can follow the guide [here](wiki/Help_centre/Tablet_conversion/en.md) for more information on how to correctly convert your old tablet driver's area into OTD's drivers.

Reviewer 2: ```suggestion
You can follow [the guide](wiki/Help_centre/Tablet_conversion) for more information on how to correctly convert your old tablet driver's area into OTD's drivers.
```

> See [Tablet conversion](/wiki/Guides/Tablet_conversion) for more information on how to correctly convert your old tablet driver's area to OpenTabletDriver's settings.

Reviewer 3: ```suggestion
See the [tablet area conversion guide](/wiki/Guides/Tablet_conversion) for more information on how to correctly convert your old tablet driver's area to OpenTabletDriver's settings.
```

> See [Tablet conversion](/wiki/Help_centre/Tablet_conversion) for more information on how to correctly convert your old tablet driver's area to OpenTabletDriver's settings.

Reviewer: ```suggestion
See [Tablet conversion](/wiki/Guides/Tablet_conversion) for more information on how to correctly convert your old tablet driver's area to OpenTabletDriver's settings.
```

> You can follow [this guide](/wiki/Help_centre/Tablet_conversion) for more information on how to correctly convert your old tablet driver's area to OpenTabletDriver's settings.

Reviewer 4: ```suggestion
See [Tablet conversion](/wiki/Help_centre/Tablet_conversion) for more information on how to correctly convert your old tablet driver's area to OpenTabletDriver's settings.
```
